database/mssql/scripts/dbscripts/PSP_PIMS_S17_00/Build/45_DML_PIMS_COUNTRY.sql

```diff
@@ -0,0 +1,20 @@
+/* -----------------------------------------------------------------------------
+Delete all data from the PIMS_COUNTRY table and repopulate.
+
+*** NOTE ***
+The PIMS_PROVINCE_STATE table must be empty due to a foreign key dependency.
+. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 
```

Comment:
I ran this specific script on my local database to update the COUNTRY ordering and it failed due to FK dependency (as stated in the note above)

How are we going to handle these changes @devinleighsmith and @dfilteau? Common assumption is that DB migrations can/must work with existing data

Reply:
![image](https://user-images.githubusercontent.com/24322224/145891064-47ea8ce3-1659-409d-8d68-47eef3efb39d.png)

Reply:
Discussed this over IM, this is not an issue.